plugins/obs-webrtc/whip-output.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <obs-module.h>
+#include <obs.hpp>
```

Comment:
Just a nitpick.
I think it is better to add this line to `whip-output.cpp` instead of `whip-output.h` because the header file does not rely on `obs.hpp`.

Reply:
Fixed! Thank you for the review @norihiro